gclient/tests/perf.rs

```diff
@@ -219,7 +219,7 @@ async fn send_messages(api: &GearApi, progs: &HashMap<&str, ProgramId>) -> Resul
 
     println!("Message count: {}", messages.len());
 
-    assert!(block_gas_limit == 0);
+    assert_eq!(block_gas_limit, 0);
```

Comment:
You may disable this test by removing this line at the moment as it occurred to be too sensitive to Wasm optimizations.